app/com/linkedin/drelephant/spark/heuristics/ConfigurationHeuristic.scala

```diff
@@ -204,7 +204,7 @@ object ConfigurationHeuristic {
       SeverityThresholds(low = MemoryFormatUtils.stringToBytes("10G"), MemoryFormatUtils.stringToBytes("15G"),
         severe = MemoryFormatUtils.stringToBytes("20G"), critical = MemoryFormatUtils.stringToBytes("25G"), ascending = true)
     val DEFAULT_SPARK_CORES_THRESHOLDS =
-      SeverityThresholds(low = 4, moderate = 6, severe = 8, critical = 10, ascending = true)
```

Comment:
Let's change for the other threshold levels as well, since it's a strict inequality.

Reply:
Do you mean to change it like :: SeverityThresholds(low = 5, moderate = 7, severe = 9, critical = 11, ascending = true)

Reply:
Yes, the original design was assuming >= rather than > for the comparison with the threshold values, so please make the change as you've suggested above.

Reply:
Added the suggested changes.